spec/golden_masters/squad_row_starcraft2.txt

```diff
@@ -0,0 +1 @@
+<div class="table-responsive"><table class="wikitable wikitable-striped roster-card" style="margin-bottom:10px"><tr><th colspan="10">Former Squad</th></tr><tr class="HeaderRow"><th>ID</th><th></th><th>Name</th><th></th><th>Join Date</th><th>Inactive Date</th><th>Leave Date</th><th>New Team</th></tr><tr class="Player"><td class="ID"><b><span><span class="starcraft-inline-player"><span class="flag">[[File:se_hd.png|36x24px|Sweden|link=]]</span>&nbsp;[[Baz|Baz]]</span></span></b></td><td></td><td class="Name"><div class="MobileStuff">(Foo Bar)</div><div class="LargeStuff">Foo Bar</div></td><td class="Position"></td><td class="Date"><div class="MobileStuffDate">Join Date:&nbsp;</div><i>2022-01-01</i></td><td class="Date"><div class="MobileStuffDate">Inactive Date:&nbsp;</div><i>2022-03-03</i></td><td class="Date"><div class="MobileStuffDate">Leave Date:&nbsp;</div><i>2022-05-01</i></td><td class="NewTeam"></td></tr></table></div>
```

Comment:
eof?

Reply:
They are auto generated and compared to, so no